monica/license.md

```diff
@@ -1 +1 @@
-View [license information](https://github.com/monicahq/monica/blob/master/LICENSE) for the software contained in this image.
+View [license information](https://github.com/monicahq/monica/blob/main/LICENSE) for the software contained in this image.
```

Comment:
Looks like this one also needs a `.md` on it :sweat_smile: 

```suggestion
View [license information](https://github.com/monicahq/monica/blob/main/LICENSE.md) for the software contained in this image.
```

Reply:
Good catch! Thanks